app.js

```diff
@@ -1,35 +1,34 @@
 const express = require("express");
 
 const { getCategories } = require("./controllers/categories.controllers");
-const { getReviewById } = require("./controllers/reviews.controllers");
+const {
+  getReviewById,
+  patchReviewById,
+} = require("./controllers/reviews.controllers");
 const { getUsers } = require("./controllers/users.controllers");
+const {
+  handlePSQLErrors,
```

Comment:
Lovely! Although your error for invalid routes doesn't seem to be tested (I may just be missing it).